src/gfx_webgl.js

```diff
@@ -1049,7 +1049,8 @@ x3dom.gfx_webgl = ( function ()
 
                 if ( sky.length != colors.length )
                 {
-                    x3dom.debug.logError( "Number of background colors and corresponding angles are different!" );
+                    x3dom.debug.logError( "Number of background colors and corresponding angles do not match.\n"
+                        + "You have to define one angle less than the count of RGB colors because the angle 0° is added automatically." );
```

Comment:
That's ok. Let's just keep logging concise. One can assume that users are familiar with the spec. in general.

Reply:
You are right, the spec tells everything in great detail: 
https://www.web3d.org/documents/specifications/19775-1/V3.3/Part01/components/enveffects.html#Backgrounds